### DISCUSSION TITLE
Ubuntu Server

### DISCUSSION THREAD

**Discussion by zigon928** [FlareSolverr/FlareSolverr#778] [Q&A]
Hey, I am struggling with seeing how, but is it possible to install this on a ubuntu server.

**Comment by ilike2burnthing** [comment]
Running the precompiled Linux binary doesn't work? https://github.com/FlareSolverr/FlareSolverr#precompiled-binaries

**Reply by ngosang** [reply]
If it does not work you can open an issue or use the Docker image.
